mapper/orm/src/main/java/org/hibernate/search/mapper/orm/outbox/impl/OutboxEventBackgroundExecutor.java

```diff
@@ -48,7 +48,14 @@ public OutboxEventBackgroundExecutor(AutomaticIndexingMappingContext mapping, Sc
 	}
 
 	public void start() {
-		executor.scheduleAtFixedRate( this::run, 0, pollingInterval, TimeUnit.MILLISECONDS );
+		executor.scheduleAtFixedRate( () -> {
+			try {
+				run();
+			}
+			catch (Throwable throwable) {
+				log.failureOnOutboxBackgroundProcessing( throwable );
+			}
```

Comment:
FYI if you switch to `SingletonTask`, you won't need this anymore, it does the reporting for you, and it correctly uses the background failure handler.